Sources/ClientRuntime/EventStream/DefaultMessageEncoderStream.swift

```diff
@@ -13,42 +13,42 @@ extension EventStream {
         let stream: AsyncThrowingStream<Event, Error>
```

Comment:
Here, the usage of `requestEncoder` is completely replaced by `marshalClosure`. This has to be done because XML serialization does not rely on Swift's Codable interface. For JSON serialization, the closure passed here will use `JSONEncoder` behind the scenes. For XML serialization, the closure passed here will use `XMLWriter` behind the scenes.